model/model.go

```diff
@@ -515,3 +521,33 @@ type PostTradeCredit struct {
 	AdjustedCreditUtilized string                      `json:"adjusted_credit_utilized"`
 	AdjustedEquity         string                      `json:"adjusted_portfolio_equity"`
 }
+
+type OnchainAddressGroup struct {
+	Id          string            `json:"id"`
+	Name        string            `json:"name"`
+	NetworkType string            `json:"network_type"`
```

Comment:
enum?

Reply:
Same response as the others. I'll add to the backlog

Reply:
Added